volatility3/framework/plugins/windows/callbacks.py

```diff
@@ -265,7 +538,13 @@ def list_bugcheck_reason_callbacks(
         layer_name: str,
         symbol_table: str,
         callback_table_name: str,
-    ) -> Iterable[Tuple[str, int, str]]:
+    ) -> Iterable[
+        Tuple[
+            str,
+            int,
+            Union[interfaces.objects.ObjectInterface, renderers.UnreadableValue],
```

Comment:
Happy with this, but feels like using the more generic `interfaces.renderers.BaseAbsentValue` might be more useful to avoid changes in the future, since it covers all of them?